buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/apiinfo/GenerateApiTxtFileTask.java

```diff
@@ -35,7 +38,7 @@
     abstract File getApiTxt();
 
     @Input
```

Comment:
I think you want `@InputFiles`, otherwise gradlew will cache by file path, not contents.